### PR TITLE
Fix missing concoct globs

### LINKED ISSUE #510: CONCOCT missing in globs in downstream processes of modules.conf
### Description of the bug

As reported by @maxibor  privately

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by jfy133**
> Wait until #511 is merged! Then test before requesting review!

closes #510 to ensure we always pick up CONCOCT related files after CONOCT itself

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,8 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Changed`
 
+- [#511](https://github.com/nf-core/mag/pull/511) Update to nf-core 2.10 `TEMPLATE` (by @jfy133)
+
 ### `Fixed`
 
+- [#512](https://github.com/nf-core/mag/pull/512) Fix missing CONCOCT files in downstream output (reported by @maxibor, fix by @jfy133)
```
Comment (jfy133) [inline CHANGELOG.md:16]:
```suggestion
- [#514](https://github.com/nf-core/mag/pull/514) Fix missing CONCOCT files in downstream output (reported by @maxibor, fix by @jfy133)
```

**Comment by jfy133** [comment]
testing...

**Comment by jfy133** [comment]
@maxibor I've tested this and it should work except DASTool does not find any bins from the CONCOCT bins with the current test data... do you have anything you can double check?

**Review by d4straub — APPROVED** [review]
LGTM

**Review by prototaxites — APPROVED** [review]
LGTM also!

**Comment by jfy133** [comment]
It should be a simple enough change so I will merge, and hopefully test with an updated full test (PR coming soon, once I've finished downloading the massive file from Australia and uploaded ti iGenomes...)  :+1:

**Pull request merged**